CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.4.0-M3 (Jun 2020)
+  * Tune GC to improve execution time performance ([`KT-19076`](https://youtrack.jetbrains.com/issue/KT-19076)).
+  * Support @Throws annotation ([`KT-34848`](https://youtrack.jetbrains.com/issue/KT-34848))
+  * Fix support of variadic block types in ObjC interop ([`KT-36766`](https://youtrack.jetbrains.com/issue/KT-36766))
```

Comment:
It's nothing like "fix support", just "handle": variadic block types are still unsupported, but at least don't cause `cinterop` crash.

Reply:
Fixed